waspc/data/Generator/templates/server/package.json

```diff
@@ -3,9 +3,11 @@
   "name": "server",
   "version": "0.0.0",
   "private": true,
-  "type": "module",
   "scripts": {
-    "start": "nodemon -r dotenv/config ./src/server.js",
+    "build": "npx tsc",
+    "start": "NODE_PATH=dist node dist/server.js",
+    "build-and-start": "npm run build && npm run start",
+    "start-nodemon": "nodemon -r dotenv/config --exec 'npm run build-and-start || exit 1'",
```

Comment:
So `start` now runs server, without watching.
`start-nodemon` starts it while watching.

Usually people expect `start` to already do watching.

I would suggest following naming:
`start` as it is, `watch` instead of `start-nodemon`.

`run` instead of `start`, `start` instead of `start nodemon`.
Although I am not sure if `run` is allowed hm.

Reply:
I chose the first one.